kubernetes_tests/test_kubernetes_pod_operator.py

```diff
@@ -112,7 +112,7 @@ def setup_tests(self, test_label):
                     "run_id": "manual__2016-01-01T0100000100-da4d1ce7b",
                     "dag_id": "dag",
                     "task_id": ANY,
-                    "try_number": "1",
+                    "try_number": "0",
```

Comment:
We shouldn't expect a try_number of 0 to be the label, should we? This might be a case where we need to have `try_number + 1` when generating the labels for the pod?

Reply:
talked with ash about this.  in this case, we're synthesizing a TI so we may as well set its try_number to be 1 so it looks more "normal"